SubcategoriesForCAP/gap/SliceCategory.gi

```diff
@@ -646,19 +648,176 @@ BindGlobal( "CAP_INTERNAL_SLICE_CATEGORY",
+            
+            C := AmbientCategory( cat );
+            
+            left_unitor := LeftUnitor( C, BaseObject( cat ) );
+            
+            #    φ           ψ
+            # i ---> i'   j ---> j'
+            #  \    /      \    /
+            # I \  / I'   J \  / J'
+            #    vv          vv
+            #    1           1
+            
+            I := Source( phi );
+            J := Source( psi );
+            
+            IP := Range( phi );
+            JP := Range( psi );
+            
+            # I⊗J: i⊗j -> 1
+            source := TensorProductOnObjectsInSliceOverTensorUnit( cat, I, J);
+            
+            # I'⊗J': i'⊗j' -> 1
+            range := TensorProductOnObjectsInSliceOverTensorUnit( cat, IP, JP );
+            
+            # φ⊗ψ: i⊗j -> i'⊗j'
+            morphism_datum := TensorProductOnMorphisms( C, UnderlyingCell( phi ), UnderlyingCell( psi ) );
+            
+            return MorphismConstructor( cat, source, morphism_datum, range );
+            
+        end );
+        
+        AddLeftUnitor( S,
+          function( cat, I )
+            local C, id_1, source, range;
+            
+            C := AmbientCategory( cat );
+            
+            #              λ_i
+            #         1⊗i ---> i
+            #           \      /
+            # id_1 ⊗ I  \    / I
+            #             v  v
+            #              1
+            
+            id_1 := ObjectConstructor( cat, IdentityMorphism( C, BaseObject( cat ) ) );
+            
+            source := TensorProductOnObjectsInSliceOverTensorUnit( cat, id_1, I );
+            
+            range := I;
+            
+            return MorphismConstructor( cat, source, LeftUnitor( C, Source( UnderlyingMorphism( I ) ) ), range );
             
         end );
         
+        AddRightUnitor( S,
+          function( cat, I )
+            local C, id_1, source, range;
+            
+            C := AmbientCategory( cat );
+            
+            #              ρ_i
+            #         i⊗1 ---> i
+            #           \      /
+            # I ⊗ id_1  \    / I
+            #             v  v
+            #              1
+            
+            id_1 := ObjectConstructor( cat, IdentityMorphism( C, BaseObject( cat ) ) );
+            
+            source := TensorProductOnObjectsInSliceOverTensorUnit( cat, I, id_1 );
+            
+            range := I;
+            
+            return MorphismConstructor( cat, source, RightUnitor( C, Source( UnderlyingMorphism( I ) ) ), range );
+            
+        end );
+        
+        AddAssociatorRightToLeft( S,
+          function( cat, I, J, K )
+            local C, source, range;
+            
+            C := AmbientCategory( cat );
+            
+            #                   α_i
+            #         i⊗(j⊗k) ---> (i⊗j)⊗k
+            #                 \      /
+            #       I⊗(J⊗K)  \    / (I⊗J)⊗K
```

Comment:
Same comment: The indentation in this line seems broken when viewed here on GitHub.